test/engine-run.test.js

```diff
@@ -13,7 +13,6 @@ describe('Engine: run', () => {
     sandbox.restore()
   })
 
-  let event = { type: 'generic' }
```

Comment:
I removed this and added unique types for each rule, so as to more easily differentiate them in a later test.